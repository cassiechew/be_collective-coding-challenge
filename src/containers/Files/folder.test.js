import { create } from 'react-test-renderer';
import React from 'react';
import Folder from './folder';

describe('snapshot test', () => {
    test('testing footer', () => {
      let tree = create(<Folder name="name" size="size" children={[]}/>);
      expect(tree.toJSON()).toMatchSnapshot();
    });
});
