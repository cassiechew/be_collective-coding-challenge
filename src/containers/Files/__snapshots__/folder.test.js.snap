// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test testing footer 1`] = `
<div
  className="container__folder"
>
  <div
    className="container__folder-block"
  >
    <img
      alt="failed"
      src="arrows.png"
    />
    <button>
      <div>
        <img
          alt="failed"
          onClick={[Function]}
          src="folder.png"
        />
         
        name
      </div>
    </button>
    <label>
      name
    </label>
  </div>
</div>
`;
